src/Components/Analyzers/src/ComponentParameterUsageAnalyzer.cs

```diff
@@ -38,8 +38,18 @@ public override void Initialize(AnalysisContext context)
                 {
                     startBlockContext.RegisterOperationAction(context =>
                     {
-                        var assignmentOperation = (IAssignmentOperation)context.Operation;
-                        var leftHandSide = assignmentOperation.Target;
+                        IOperation leftHandSide;
+
+                        if (context.Operation is IAssignmentOperation assignmentOperation)
+                        {
+                            leftHandSide = assignmentOperation.Target;
+                        }
+                        else
+                        {
+                            var incrementOrDecrementOperation = (IIncrementOrDecrementOperation)context.Operation;
+                            leftHandSide = incrementOrDecrementOperation.Target;
```

Comment:
this is actually a great place to earn #PranavPoints by using a switch expression.

Reply:
hahah, so true. Going to leave for now so I don't have to wait for a rebuild though.